Client/src/Pages/Infrastructure/index.jsx

```diff
@@ -1,5 +1,72 @@
-import { Box, Button } from "@mui/material";
+import {
+	Box,
+	Button,
+	IconButton,
+	Paper,
+	Stack,
+	Table,
+	TableBody,
+	TableCell,
+	TableContainer,
+	TableHead,
+	TableRow,
+	// TablePagination,
+	// Typography,
+} from "@mui/material";
 import { Heading } from "../../Components/Heading";
+import { useTheme } from "@emotion/react";
+import Greeting from "../../Utils/greeting";
+import Breadcrumbs from "../../Components/Breadcrumbs";
+import { StatusLabel } from "../../Components/Label";
+import { Gauge } from "../../Components/Charts/Gauge";
+import GearIcon from "../../Assets/icons/settings-bold.svg?react";
+import CPUChipIcon from "../../Assets/icons/cpu-chip.svg?react";
```

Comment:
Yeah, I changed it but git does not recognize that because there were no changes to the folder. I was just making capitalization for assets match the other folders inside of src